Src/Plugins/Security/SEC101_015.AkamaiCredentialsValidator.cs

```diff
@@ -14,15 +15,84 @@ namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security
 {
     public class AkamaiCredentialsValidator : DynamicValidatorBase
     {
-        protected override IEnumerable<ValidationResult> IsValidStaticHelper(IDictionary<string, FlexMatch> groups)
+        internal static HttpRequestMessage GenerateRequestMessage(string id,
+                                                                  string host,
+                                                                  string secret,
+                                                                  string resource,
+                                                                  string scanIdentityGuid,
+                                                                  DateTime datetime)
+        {
+            string timestamp = datetime.ToString("yyyyMMdd'T'HH:mm:ss+0000");
+            using var request = new HttpRequestMessage(HttpMethod.Get, $"{host}/ccu/v2/queues/default");
+            string requestData = GetRequestData(request.Method.ToString(), request.RequestUri);
+            string authData = GetAuthDataValue(id, resource, timestamp, scanIdentityGuid);
+            string authHeader = GetAuthorizationHeaderValue(secret, timestamp, authData, requestData);
+            request.Headers.Add("Authorization", authHeader);
+
+            return request;
+        }
+
+        internal static string GetRequestData(string method,
+                                              Uri uri)
```

Comment:
not all methods should follow this style.
only the ones that gets too long.

in this case, you can keep in one line.